Software/backend/database/MongoDB/Main.md

```diff
@@ -0,0 +1,42 @@
+<!-- press ctrl+K then V to open a preview of the MarkDown file  -->
+# MonogoDB
+MongoDB is a cross-platform document-oriented database program. 
+Classified as a NoSQL database program, MongoDB uses JSON-like documents with optional schemas. 
+MongoDB is developed by MongoDB Inc. and licensed under the Server Side Public License (SSPL).
+
+# Table Of Contents
+
+* [Intro](#Intro) 
+* [Platform](#Platform)
+* [Repositories](#Repositories)
+* [Docs](#Docs)
+
+# Intro
+* Document databases differ from relational databases in how they organize data as well as the query languages they support. 
+Document databases are usually categorized under the NoSQL umbrella term.
+
+# Platform
+* The recommended paltfrom to use is [MongoDB Atlas](https://www.mongodb.com/cloud/atlas)
+* [Atlas Guide](./AtlasGuide)
+* [Combine with Node.js](./AtlasWithNode)
+
+# Repositories
+* Mongoose 
+``` Javascript
+npm i mongoose --save
+
+```
+
+* MongoDB 
+``` Javascript
+npm i mongodb --save
+
+```
+
+
+# Docs
+* [Documents](https://docs.mongodb.com/manual/core/document/) [Collections](https://docs.mongodb.com/manual/core/databases-and-collections/)
```

Comment:
if you want the description to be in a new line, add two spaces after the link (.md is weird like that).
maybe make some seperation between the two links, as they it looks confusing